### ISSUE TITLE
[BUG] Github saver shows incorrect default branch and can fail with 404.

### ISSUE THREAD

**Issue by saqimtiaz** [Jermolene/TiddlyWiki5#5317]
With the recent changes to the Github saver https://github.com/Jermolene/TiddlyWiki5/commit/3b75297168d411bb9e0ed117ece5cab16eba6a9f and https://github.com/Jermolene/TiddlyWiki5/commit/8cd13e2f89ab85b47794a4f6198fbb1418775704 we overlooked needing to change the default placeholder for the branch name shown in the corresponding input field in the saver UI: https://github.com/Jermolene/TiddlyWiki5/blob/master/core/ui/ControlPanel/Saving/GitHub.tid#L13

So when the user has not manually filled in a branch to use, the placeholder shown is "master" but the saver uses the new default "main", resulting in a 404.

The interim fix for users is fortunately easy, just empty and re-fill the branch name in the Github Saver UI in the Control Panel.

**Comment by seanpascoe** [comment]
> So when the user has not manually filled in a branch to use
further than that, when the user manually enters "master", it still tries to save to the "main" branch. This one got me until I looked at the network request in devtools. I switched to the main branch in git and now it works fine. I am totally new to this project but I wonder why the error message isn't displayed in the UI. All I got was a "404" alert in the browser.
